nifi-system-tests/nifi-system-test-suite/src/test/resources/conf/default/bootstrap.conf

```diff
@@ -27,7 +27,7 @@ java.arg.3=-Xmx512m
 
 java.arg.14=-Djava.awt.headless=true
 
-#java.arg.debug=-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=8002
+java.arg.debug=-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=8002
```

Comment:
It looks like this change should not be included, so that remote debugging is disabled by default in the system tests.